packages/liveblocks-react-ui/src/slate/utils/is-empty.ts

```diff
@@ -1,10 +1,39 @@
 import type { Descendant } from "slate";
 import { Editor as SlateEditor, Text as SlateText } from "slate";
 
+import { isEmptyString } from "./is-empty-string";
+
 export function isEmpty(editor: SlateEditor, children: Descendant[]) {
-  return (
-    children.length <= 1 &&
-    !SlateText.isText(children[0]) &&
-    SlateEditor.isEmpty(editor, children[0])
-  );
+  // No children
+  if (children.length === 0) {
+    return true;
+  }
+
```

Comment:
You could remove this case, because it's already covered by the loop that won't do any looping.

Reply:
😅 Thanks for the catch!